sklearn/__init__.py

```diff
@@ -96,13 +96,12 @@ def setup_module(module):
     """Fixture for the tests to assure globally controllable seeding of RNGs"""
     import os
     import numpy as np
-    import random
 
     # It could have been provided in the environment
     _random_seed = os.environ.get('SKLEARN_SEED', None)
     if _random_seed is None:
         _random_seed = np.random.uniform() * (2 ** 31 - 1)
     _random_seed = int(_random_seed)
     print("I: Seeding RNGs with %r" % _random_seed)
-    np.random.seed(_random_seed)
-    random.seed(_random_seed)
+    rng = np.random.RandomState(_random_seed)
+    rng.seed(_random_seed)
```

Comment:
Hah, I was not aware this was happening. The `setup_module` function was last touched 4 years ago. Maybe let's keep it unchanged for now, and see in some other PR if we want to change this.

Reply:
Thank You @rth .
I have reverted this bit and will be fixing places where the tests are breaking. (the tests might probably be failing because of using RandomState.random instead of RandomState.random_sample)